langserver/util.go

```diff
@@ -22,3 +22,7 @@ func PathTrimPrefix(s, prefix string) string {
 	}
 	return strings.TrimPrefix(s, prefix)
 }
+
+func PathEqual(a, b string) bool {
```

Comment:
Does this need to be exported?

Since it's exported, it should be documented per [our style guide](https://github.com/sourcegraph/sourcegraph/blob/master/docs/style.md#sourcegraph-style-guide) which simply defers to https://github.com/golang/go/wiki/CodeReviewComments#doc-comments on this.